src/app/(proper_react)/redesign/(authenticated)/user/welcome/Onboarding.test.tsx

```diff
@@ -169,22 +140,57 @@ it("explainer dialog shows on step 2", async () => {
 it("confirm dialog is showing on step 2", async () => {
   const user = userEvent.setup();
   const ComposedOnboarding = composeStory(Onboarding, Meta);
-  const { container } = render(<ComposedOnboarding stepId="enterInfo" />);
+  render(<ComposedOnboarding stepId="enterInfo" />);
 
   expect(screen.queryByRole("dialog")).not.toBeInTheDocument();
-  populateEnterInfoForm(container);
+
+  const firstNameField = screen.getByLabelText("First name*");
+  const lastNameField = screen.getByLabelText("Last name*");
   const proceedButton = screen.getByRole("button", {
     name: "Find exposures",
   });
+  const dobField = screen.getByLabelText("Date of birth*");
+  await user.type(firstNameField, "User");
+  await user.type(lastNameField, "Name");
+  await user.keyboard("[Tab]Tu[ArrowDown][Enter][Tab]");
```

Comment:
That’s cool!

Reply:
Actually it was a major PITA to find out how to get the suggestion list to show up in test renders, and this was the only way I found to do it :sweat_smile: